fastlane_bot/config/constants.py

```diff
@@ -8,5 +8,9 @@
 FLASHLOAN_FEE_MAP = {
     "ethereum": 0,
     "coinbase_base": 0,
-    "fantom": 0.0003
+    "fantom": 0.0003,
+    "mantle": 0,
 }
+CARBON_V1_NAME = "carbon_v1"
+CARBON_V1_FORK1_NAME = "altered_carbon"  #TODO CHANGE BEFORE MERGING
+DEFAULT_ETHEREUM_EXCHANGES = "carbon_v1,bancor_v3,bancor_v2,bancor_pol,uniswap_v3,uniswap_v2,sushiswap_v2,balancer,pancakeswap_v2,pancakeswap_v3,altered_carbon"
```

Comment:
it strikes me as strange to see a list dealing specifically with ethereum and no similar lists for other networks